unpackaged/config/dev/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
     <types>
-        <members>*</members>
+        <members>%%%NAMESPACE%%%HEDA</members>
```

Comment:
Is this CustomApp identical to the one we package in terms of tabset, currently? I would expect a deployment to fail in a managed context if the tabset differs. Are you seeing different behavior?

Reply:
It is. Also we do not see a deployment failure. App tabs can be appended to. I haven't tested for what occurs when we try to remove tabs, but I did want to change this to be explicit. We've been running this for a long time in the [QA config.](https://github.com/SalesforceFoundation/EDA/blob/main/unpackaged/config/qa/package.xml)

The only reason I am not recommending we make a different app when performing dev/qa work is for ease of lqa and the objects being in "one place"

Reply:
Sounds good. I'll go ahead and approve, just calling it out as a potential risk if there are future modifications. (Fairly sure it will fail if tabs are removed).